Frameworks/include/CGIWICBitmap.h

```diff
@@ -187,17 +187,36 @@ class CGIWICBitmap : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::Runtime
                                          _Out_writes_all_(cbBufferSize) BYTE* buffer) {
         RETURN_HR_IF_NULL(E_POINTER, buffer);
 
+        const WICRect fullRect = { 0, 0, m_width, m_height };
```

Comment:
That's fine?

---
In reply to: [92295863](https://github.com/Microsoft/WinObjC/pull/1529#discussion_r92295863) [](ancestors = 92295863)